### ISSUE TITLE
Source config files from ~/.tmux

### ISSUE THREAD

**Issue by jamesottaway** [jamesottaway/tmux-up#5]
This would be a good place to store non-workspace related configurations.

For example, I can see myself having a `bootstrap` session for keeping the following tools up-to-date:
- `nix`
- `fresh`
- `homebrew`
- `homebrew-cask`

**Comment by poxar** [comment]
We might want to change the naming scheme for those default configurations. Right now they get a name in the form of `pwd/sessionname`. Wouldn't something like `std/sessionname` or even just `sessionname` be better?

**Comment by jamesottaway** [comment]
> We might want to change the naming scheme for those default configurations.

Yeah, I think we should drop the prefix altogether, so just have `foo` instead of `tmux/foo`.
